src/app/app.js

```diff
@@ -0,0 +1,83 @@
+let app = angular.module('CUSTOM_NAME_OF_THE_APP', ['ngMockE2E', 'ngAnimate', 'ui.router', 'anim-in-out', 'ui.bootstrap', 'ngSanitize', 'pascalprecht.translate', 'angular-loading-bar']);
+let languages = ['en', 'fr'];
+
+(function () {
```

Comment:
I gave a try with Babel 6 but it seems that it just let the braces. I opened a topic about that on Babel :  [http://discuss.babeljs.io/t/iife-block-only/211](http://discuss.babeljs.io/t/iife-block-only/211)